Library/Homebrew/sandbox.rb

```diff
@@ -8,11 +8,12 @@ class Sandbox
   ].freeze
 
   def self.available?
-    OS.mac? && File.executable?(SANDBOX_EXEC)
+    OS.mac? && OS::Mac.version >= "10.6" && File.executable?(SANDBOX_EXEC)
```

Comment:
Does that file exist and is executable on e.g. 10.5 but just doesn't work? If not I think we can leave it as-is.